utils/gen-snapshot-tests.js

```diff
@@ -44,6 +44,9 @@ function generateTestFile(root, examples, componentFileMap) {
         "//   2. Run `yarn run gen-snapshot-tests`.",
         `import React from "react";`,
         `import renderer from "react-test-renderer";`,
+        ``,
+        `// Mock react-dom as jest doesn't like findDOMNode.`,
+        `jest.mock("react-dom");`,
```

Comment:
This is the actual change. The rest of the changes are the auto-generated output.